v4/integrations/nrgin/go.mod

```diff
@@ -1,10 +1,10 @@
-module github.com/newrelic/go-agent/v3/integrations/nrgin
+module github.com/newrelic/go-agent/v4/integrations/nrgin
 
 // As of Dec 2019, the gin go.mod file uses 1.12:
 // https://github.com/gin-gonic/gin/blob/master/go.mod
 go 1.12
 
 require (
-	github.com/gin-gonic/gin v1.4.0
-	github.com/newrelic/go-agent/v3 v3.0.0
+	github.com/gin-gonic/gin v1.5.0
```

Comment:
When running the nrgin tests with gin v1.4.0 the following error was raised. I saw this error in the github actions logs, when running the tests locally in docker, and when running the tests locally outside of docker.

```
##[error]../../../../../../../pkg/mod/github.com/gin-gonic/gin@v1.4.0/binding/msgpack.go:12:2: ambiguous import: found package github.com/ugorji/go/codec in multiple modules:
	github.com/ugorji/go v1.1.4 (/home/runner/work/go-agent/go-agent/pkg/mod/github.com/ugorji/go@v1.1.4/codec)
	github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8 (/home/runner/work/go-agent/go-agent/pkg/mod/github.com/ugorji/go/codec@v0.0.0-20181204163529-d75b2dcb6bc8)
```

The error did not occur when running the nrgin tests against the v3 agent.

I was unable to track down the exact reason why this failure was happening. The `github.com/gin-gonic/gin` package requires `github.com/ugorji/go v1.1.4 ` in its go.mod file. I checked everywhere to try to find out which package was requiring the `github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8` but couldn't find it. Go's tooling for this wasn't very helpful.

After spending about an hour and a half on trying to find the cause, I decided to switch to solutions. For this I found that simply upgrading `github.com/gin-gonic/gin` to v1.5.0 did the trick. v1.4.0 of gin was released on May 7, 2019 and v1.5.0 was released on Nov 27, 2019. The most recent version of gin is v1.6.3. You can see the full [release notes here](https://github.com/gin-gonic/gin/releases).